server/db/schemas/patients.sql

```diff
@@ -1,5 +1,7 @@
 CREATE TABLE patients (
     patient_id INT,
+    doctor_id INT,
```

Comment:
Any better name for this? that represents this is the doctor id of the patients doctor not the doctor id of the patient itself, which wouldnt make sense. If no better name then just leave it its fine

Reply:
what about assigned_doctor_id?

Reply:
i think thats the name i put in the ticket but idk lol cause that sounds like it the doctor id assigned to the patient which doesnt sounds right either. Up to you to be honest, idm either way